eduNEXT/platform-global-teacher-campus
# feat: added new rejection list endpoint

This PR adds the GET endpoint to list the rejection reasons created.

**How to test**

1. Make an API call as following local.overhang.io:8000/plugin-cvw/api/v1/rejection-reasons (you can use the studio domain as well if you instaleld the plugin inside the CMS)

2. You should see a response like this:

```json
[
    {
        "id": 1,
        "name": "rejection1"
    },
    {
        "id": 2,
        "name": "rejection 2"
    }
]
```
